bundles/ranvier-vendors/behaviors/npc/vendor.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+
+module.exports = (srcPath) => {
+  const Logger = require(srcPath + 'Logger');
+  const B      = require(srcPath + 'Broadcast');
+  return  {
+    listeners: {
+      playerEnter: state => function (config, player) {
+        if (config.enterMessage) {
+          B.sayAt(player, config.enterMessage);
```

Comment:
I think it'd be cool for NPC messages to the player to use the `say` channel. `state.ChannelManager.get('say').send(state, this, config.enterMessage);` So that coloring/format is consistent between player say/NPC say.